definitions/marts/looker_studio_marts/npq/ls_npq_tsf_application_detail.sqlx

```diff
@@ -3,13 +3,18 @@ config {
     assertions: {
         uniqueKey: ["application_id"]
     },

```

Comment:
I think we should cluster by cohort, you can see how to do that in the npq_enrolment code in dataform